src/main/java/duke/io/Ui.java

```diff
@@ -0,0 +1,106 @@
+package duke.io;
+import duke.task.Task;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Ui - Handles interactions with the user, including input and output.
+ */
+public class Ui {
+    private static final String LINE_SEPARATOR = "____________________________________________________________";
```

Comment:
Good job remembering to capitalise the constant name! And idea to create a printline() method is quite cool and neat too :D